src/store/modules/keycodes/quantum.js

```diff
@@ -31,7 +31,14 @@ export default [
     title: 'Manually enter any QMK keycode'
   },
 
-  { label: 'Layer and Layer Tap functions', width: 'label' },
+  {
+    label: 'Layer and Layer Tap functions',
+    icon: 'exclamation-triangle',
+    iconClass: 'warning',
+    width: 'label',
+    title:
+      'Note: Layer keys override the position on the target layer to TRNS to avoid layer lock'
```

Comment:
```suggestion
      'Note: Layer keys overwrite the key at the same position on the target layer to KC_TRNS to avoid layer lock'
```

"Override" implies the key will return to its original assignment if the layer key is removed.

Reply:
Sorry that makes no sense unless we allow undo.

Reply:
Uh, it *does* make sense *because* undo is not supported (yet).

Reply:
I do like the overwrite better though. I will need to test to see if it's too verbose for the screen. It's quite long already and on a contricted width it may be too wordy

Reply:
Agree to disagree. But thank you for the input.

Reply:
I'm confused as to what we're disagreeing on... you say you like "overwrite" better, which is exactly what the suggestion does.

Reply:
For clarity, my comment below the suggestion is about the current behaviour, where if you remove a MO(), the transparent key above it is not also removed or reverted back to what it was before the placement of the MO() turned it into KC_TRNS. I call that "overwriting", not "overriding".

Reply:
Thanks very much.